usr/bin/steamos-session

```diff
@@ -33,6 +33,18 @@ CONFIG_FILE="$CONFIG_PATH/steamos-compositor-plus"
 
 STEAMCMD="steam -tenfoot -steamos -fulldesktopres"
 COMPOSITORCMD="steamcompmgr"
+
+# Source environment from ~/.config/environment.d
+for i in "~/.config/environment.d/*.conf" ;
+do
+    if [ -f "${i}" ]; then
+        set -a
```

Comment:
what is `set -a` and `set +a` doing?

Reply:
`set -a` makes any variable set as `VAR=VAL` automatically exported. That way every file that hast the expected syntax is exported. `set +a` disables that behavior.